src/SplitScreen/SplitScreen.js

```diff
@@ -0,0 +1,276 @@
+
+    // update UI & alert component parent of change
+    this.handleMapChange(visibleMapCount - 1)
+
+    if (visibleMapCount - 1 === 2) this.setState({ startPosition: window.innerWidth / 2 })
+    onMapRemoved(mapIndex)
+  }
+
+  addMap = () => {
+    const { onMapAdded, maps, visibleMapCount } = this.props
+    const nextMapIdx = visibleMapCount
+    const prevMap = maps[nextMapIdx - 1]
+    const mapToAdd = maps[nextMapIdx]
+    const prevMapLayer = prevMap.getLayerGroup().getLayers().item(0)
+
+    mapToAdd.setVisibleState(true)
+    mapToAdd.setSyncedState(true)
+
+    mapToAdd.getLayers().setAt(0, prevMapLayer)
+
+    mapToAdd.getView().setCenter(prevMap.getView().getCenter())
+    mapToAdd.getView().setZoom(prevMap.getView().getZoom())
+
+    // first de-sync everything (this prevents double binding sync events)
+    maps.map(m => this.desyncMapEvents(m.getView()))
+
+    // now get the syncable maps and then sync them
+    maps.filter(m => m.getSyncedState()).map(m => this.syncMapEvents(m.getView()))
+
+    // update UI & alert component parent of change
+    this.handleMapChange(visibleMapCount + 1)
+
+    if (visibleMapCount + 1 === 2) this.setState({ startPosition: window.innerWidth / 2 })
+    onMapAdded(nextMapIdx)
+  }
+
+  toggleSyncMap = (i) => {
+    const { maps, toggleSyncMap } = this.props
+    const currentSyncState = maps[i].getSyncedState()
+
+    // set the map sync state to the inverse
+    maps[i].setSyncedState(!currentSyncState)
+
+    // first de-sync everything (this prevents double binding sync events)
+    maps.map(m => this.desyncMapEvents(m.getView()))
+
+    // now get the syncable maps and then sync them
+    maps.filter(m => m.getSyncedState()).map(m => this.syncMapEvents(m.getView()))
+
+    // update UI & alert component parent of change
+    this.props.forceUpdate()
+    toggleSyncMap()
+  }
+
+  render () {
+    const { startPosition } = this.state
+    const { translations, maps, syncedState, visibleState, visibleMapCount } = this.props
+    const disabled = false
+
+    this.handleMapChange(visibleMapCount, false)
+
+    return (
+      <Container>
+        {disabled
+          ? <div className='alert alert-warning' role='alert'>{translations['geokit.SplitScreen.disabled']}</div>
+          : null
+        }
+        <MapDisplayContainer disabled={disabled}>
+          {visibleState.map((visible, i) => {
+            const grow = i === 0 && visibleMapCount === 3
+
+            if (visible) {
+              return <MapDisplayElement
+                key={i}
+                index={i}
+                map={maps[i]}
+                grow={grow}
+                mapNumber={i + 1}
+                synced={syncedState[i]}
+                toggleSyncMap={this.toggleSyncMap}
+                disabled={disabled}
+                translations={translations} />
+            }
+          })}
+        </MapDisplayContainer>
+        {startPosition && visibleMapCount === 2 &&
```

Comment:
What about when `visibleMapCount` gets beyond 3 (e.g. 4 or other even numbers thereafter)? Should the be `visibleMapCount % 2 === 0` math?